examples/set-new-validators/index.ts

```diff
@@ -0,0 +1,115 @@
+import { Chain, createPublicClient, http } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { arbitrumSepolia } from 'viem/chains';
+import {
+  createRollupFetchTransactionHash,
+  createRollupPrepareTransactionReceipt,
+  rollupAdminLogicPublicActions,
+} from '@arbitrum/orbit-sdk';
+import { sanitizePrivateKey } from '@arbitrum/orbit-sdk/utils';
+import { config } from 'dotenv';
+config();
+
+if (typeof process.env.ROLLUP_ADDRESS === 'undefined') {
+  throw new Error(`Please provide the "ROLLUP_ADDRESS" environment variable`);
+}
+
+if (typeof process.env.ROLLUP_OWNER_PRIVATE_KEY === 'undefined') {
+  throw new Error(`Please provide the "ROLLUP_OWNER_PRIVATE_KEY" environment variable`);
+}
+
+if (typeof process.env.NEW_VALIDATOR_ADDRESS === 'undefined') {
+  throw new Error(`Please provide the "NEW_VALIDATOR_ADDRESS" environment variable`);
+}
+
+function getBlockExplorerUrl(chain: Chain) {
+  return chain.blockExplorers?.default.url;
+}
+
+// set the parent chain and create a public client for it
+const parentChain = arbitrumSepolia;
+const parentChainPublicClient = createPublicClient({
+  chain: parentChain,
+  transport: http(),
+}).extend(
+  rollupAdminLogicPublicActions({
+    rollup: process.env.ROLLUP_ADDRESS as `0x${string}`,
+  }),
+);
+
+// load the deployer account
+const deployer = privateKeyToAccount(sanitizePrivateKey(process.env.ROLLUP_OWNER_PRIVATE_KEY));
+
+async function main() {
+  //
+  // Parent chain
+  // ------------
+  //
+  // get the core contracts
+  console.log('Fetching UpgradeExecutor address in the parent chain...');
+  const transactionHash = await createRollupFetchTransactionHash({
+    rollup: process.env.ROLLUP_ADDRESS as `0x${string}`,
+    publicClient: parentChainPublicClient,
+  });
+  const transactionReceipt = createRollupPrepareTransactionReceipt(
+    await parentChainPublicClient.getTransactionReceipt({ hash: transactionHash }),
+  );
+  const coreContracts = transactionReceipt.getCoreContracts();
+  console.log(`UpgradeExecutor address: ${coreContracts.upgradeExecutor}`);
+
+  // here we just set one new validator, so the array length is 1
+  const newValidators = [process.env.NEW_VALIDATOR_ADDRESS as `0x${string}`];
```

Comment:
https://viem.sh/docs/glossary/types#address
```suggestion
  const newValidators = [process.env.NEW_VALIDATOR_ADDRESS as Address];
```